lib/twilio/errors/generated.ts

```diff
@@ -252,6 +252,85 @@ export namespace GeneralErrors {
   }
 }
 
+export namespace MalformedRequestErrors {
+  export class MalformedRequestError extends TwilioError {
```

Comment:
Did you run this on the browser? What does the error object look like? Please try to invoke it and post the object here.

Reply:
<img width="812" alt="Screen Shot 2022-10-21 at 1 49 57 PM" src="https://user-images.githubusercontent.com/12516091/197286898-26d31488-080c-4172-9b70-0ee1ac34c052.png">